### DISCUSSION TITLE
Error message when logging in on example project

### DISCUSSION THREAD

**Discussion by cordial** [gladly-team/next-firebase-auth#93] [Q&A]
**Describe the bug**
Error Message displayed when trying to login new user with the /example project 
**Version**
^0.13.0-alpha.0 (also tried 0.12.1)

**To Reproduce**
1. Clone project and run the example folder project (using an existing firebase project and keys)
2. Open localhost:3000
3. click login in header
4. Register new user by filling out email address and password
5. Signout of new account
6. Try and login with new account, you get the following error message - 

Error: Problem getting a refresh token: {"error":{"code":400,"message":"INVALID_CUSTOM_TOKEN","errors":[{"message":"INVALID_CUSTOM_TOKEN","domain":"global","reason":"invalid"}]}}
    at /Users/davidcarter/work/next-firebase-auth/example/node_modules/next-firebase-auth/build/index.node.js:2:10542
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/davidcarter/work/next-firebase-auth/example/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/davidcarter/work/next-firebase-auth/example/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

**Expected behavior**
I expect the user to login without an error. 

**Additional context**
tried on Brave and Chrome

**Comment by kmjennison** [comment]
This is most likely an implementation error. Please see some suggestions on debugging here: 
https://github.com/gladly-team/next-firebase-auth/issues/90#issuecomment-784549144

Feel free to share more info if you're unable to get things working, and please update here if you identify the problem.
